### ISSUE TITLE
Prediction Step: After training with calms21: unable to input new dataset

### ISSUE THREAD

**Issue by zhenyuan992** [YttriLab/A-SOID#29]
i have successfully trained asoid on the calms21 dataset, but now in the next step, I am unable to input new dataset.
may I know what should I do here? 

![image](https://user-images.githubusercontent.com/12291964/234501470-2937d421-a28e-4bfd-b4be-87586d41cc9a.png)

or where should I get the DLC of the calms21 dataset?
thanks!

**Comment by JensBlack** [comment]
Hi again,

I don't know what you are trying to do here but let me ellaborate on the "predict" option and maybe this helps you.

### The prediction step

You can upload pose estimation files (e.g. from DLC or SLEAP) to use the previously trained classifier (from the active learning step). This is meant to predict new data, so it does not allow to throw in the CalMS21 data just like this. Is that what you are trying to do?

1. If you want to know the f1 score/ performance of the model on the dataset, you can go back to the ``active learning`` tab and check out the results. The ``--`` lines indicate the performance on the entire dataset, and the other the performance of the iteration.

2. If you want to force A-SOiD to predict a test set for external verification, the current version allows you to input DLC- or SLEAP-style files, so you would have to convert some of the files from the CALMS21 dataset into that format and throw them in manually.

3. If you want to predict on new data, you can use the model to predict from the same or different pose estimation model origins (e.g., DLC) by just selecting them (excluding CALMS21). Under the hood, A-SOiD transforms both input types into a common format so that both can be used for prediction, even if the original training data was from the other source.

> Note that this also allows you to use a CALMS21-trained model to predict non-dataset data, although the performance might vary, if your own data is different from the dataset. In that case, the test set (f1 score) does not give you a reasonable estimate of the model's performance, and you would have to verify this further.

**Comment by zhenyuan992** [comment]
i am also trying bsoid (fully unsupervised classification) but its giving me a lot of problems.
these problems are present even when using the dataset within the bsoid github

**Comment by JensBlack** [comment]
@zhenyuan992 Concerning B-SOiD issues, if you haven't done so already, I urge you to create an issue in that repository to keep both projects separated (on a development level). @runninghsus

Concerning labeling: You do not need to use Boris for labeling if you have an alternative solution; I am happy to incorporate this into our import structure. This issue has been happening enough that I am considering to create a step-by-step guide on exporting BORIS files into an ASOiD fitting format. Let me know if this would be interesting for you.

**Comment by JensBlack** [comment]
I am closing this issue as it seems that the problem has been addressed. Please feel free to open it again
